metron-platform/metron-parsers/src/main/config/zookeeper/parsers/asa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "parserClassName": "org.apache.metron.parsers.asa.BasicAsaParser",
+  "sensorTopic": "asa",
+  "parserConfig": {
+    "deviceTimeZone": "UTC-05:00"
```

Comment:
Yes, absolutely. I'll remove it. I left this in as an example. If no `deviceTimeZone` is provided, the code will default to UTC.